release-notes.txt

```diff
@@ -1,3 +1,514 @@
+RIOT-2019.07 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2019.07 release includes a number of new features including many new
+boards and cpu, riotboot added to many new and old boards, USB is now available,
+BLE improvements, Ethernet on stm32 platforms, as well as many bug fixes and
+documentation updates.  Testing has also improved with both On-Target Testing
+increasing and now Hardware Assisted Automated Tests being run.
+
+About 300 pull requests with about 659 commits have been merged since the last
+release and about 50 issues have been solved. 26 people contributed with code
+in 106 days. Approximately 1377 files have been touched with 181993 insertions
+and 19668 deletions.
+
+
+Notations used below:
+=====================
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+System libraries
+----------------
+    + Add OCB encryption mode
+    + sys/shell: add loramac shell command
+    * Fletcher16: extend with multi-part functions
+    + USBUS: Initial work towards an USB stack
+    + usbus: Initial simple auto init structure
+    * sys: make uart_stdio RX optional
+    + sys/event: add event_wait_until()
+    + sys/bluetil: add bluetil_addr_from_str()
+    + usbus: Add CDC-ECM (Ethernet Control Model) function
+    * usbus: simplify adding entry to list
+    * sys/stdio_ethos: replace USE_ETHOS_FOR_STDIO by stdio_ethos pseudomodule
+
+Networking
+----------
+    * gnrc_ipv6_nib: add address from netif to address validation timer
+    + netdev_ieee802154: add txpower and page
+    * gnrc_sixlowpan_frag: allow send of multiple datagrams simultaneously
```

Comment:
Sorry I wasn't clear on that, but I also meant this line with https://github.com/RIOT-OS/RIOT/pull/11901#discussion_r306795543

Reply:
Thanks for clarifying!